deploy/cloud/helm/dynamo-platform-values.yaml

```diff
@@ -37,9 +37,9 @@ dynamo-operator:
       gateway: ${ISTIO_GATEWAY}
     ingressHostSuffix: ${DYNAMO_INGRESS_SUFFIX}
     dockerRegistry:
-      server: ${PIPELINES_DOCKER_SERVER}
-      username: ${PIPELINES_DOCKER_USERNAME}
-      password: ${PIPELINES_DOCKER_PASSWORD}
+      server: ${DYNAMOGRAPH_DOCKER_SERVER}
+      username: ${DYNAMOGRAPH_DOCKER_USERNAME}
+      password: ${DYNAMOGRAPH_DOCKER_PASSWORD}
     imageBuildEngine: buildkit
     virtualServiceSupportsHTTPS: ${VIRTUAL_SERVICE_SUPPORTS_HTTPS}
 
```

Comment:
I think there should be some more helm chart updates, I see pipeline mentioned in the following files: 
* deploy/helm/chart/Chart.yaml
* deploy/cloud/helm/platform/components/operator/values.yaml
* deploy/cloud/helm/platform/components/operator/Chart.yaml
* deploy/cloud/helm/platform/values.yaml

@julienmancuso would changing it in these files cause any issue?